src/ts/tinymce.ts

```diff
@@ -475,28 +475,23 @@ export function getTinymceBaseConfig(page: string): object {
       },
     ],
     toolbar_sticky: true,
-    // specifying custom CSS for properly rendering MathJax in TinyMCE instance
-    // CSS properties obtained from inspecting the inline CSS for MathJax elements displayed in view mode (MathJax 3.2.2)
-    content_style: 'mjx-assistive-mml { position: absolute !important; top: 0px; left: 0px; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0px 0px 0px !important; border: 0px !important; display: block !important; width: auto !important; overflow: hidden !important; user-select: none; }' +
-      'g[data-mml-node="merror"] > rect[data-background] { fill: yellow; stroke: none; }' +
-      'g[data-mml-node="merror"] > g { fill: red; stroke: red; }',
     // render MathJax for TinyMCE preview
     init_instance_callback: (editor) => {
       editor.on('ExecCommand', (e) => {
         if (e.command == 'mcePreview') {
           // declaration as iFrame element required to avoid errors with getting srcdoc property
           const iframe = (document.querySelector('iframe.tox-dialog__iframe') as HTMLIFrameElement);
           if (iframe) {
-            const htmlString = iframe.srcdoc;
-            // reload iFrame container text
-            iframe.srcdoc = '';
-            iframe.srcdoc = htmlString;
-            // parse text with MathJax once iFrame has been fully loaded
             iframe.onload = () => {
-              // parse text only when iFrame is finished being loaded
-              if (iframe.contentDocument && iframe.contentDocument.body) {
-                MathJax.typesetPromise([iframe.contentDocument.body]);
-              }
+              const tinyDiv = document.createElement('div');
+              tinyDiv.setAttribute('id', 'tinymce-preview');
```

Comment:
is this id important? Can we get rid of it? If it is important, explain why in a comment above